src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2230,7 +2241,7 @@ def _download(
         target_name,
         out_file_name=None,
         chunk_size=DEFAULT_CHUNKSIZE,
-        progress_bar=None,
+        progress_bar=False,
```

Comment:
The default value was ``None`` to leave it up to the underlying function ``save_chunks_to_file``  the setting up of that parameter. But yeah, it is better like that.